### ISSUE TITLE
PositionInSet, SizeofSet and Level property is not defined for Tree items in left pane under 'Workbook' tab item

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#357]
### Steps to Reproduce
1. Open "Xamarin Workbook"
2. Products window is opened.
3. Navigate to the "wpf "icon and select "create workbook".
4. Navigate to tree items present in left side pane under 'workbook' tab item and  Observe Inspect property

### Expected Behavior
PositionInSet, SizeofSet and level should be properly defined for Tree items 
### Actual Behavior
PositionInSet, SizeofSet and level property is not defined for Tree items
Narrator Behavior: Narrator is announcing as 'Xamarin.Interactive.Workbook.Structure.WorkbookTitledNode, 4 of 4, collapsed, selected'
NVDA behavior:NVDA is announcing as ''Tree view
Level 2 Xamarin.Interactive.Workbook.Structure.WorkbookTitledNode,Collapsed''

### User Impact:
User who rely on screen reader will not not be able to know position and Size of Tree items

### Screen-Shots: 
![positionsize_treeitem_inspect](https://user-images.githubusercontent.com/22956130/40724289-5b455186-643e-11e8-9af8-592300fd4e80.png)
![positionsize_treeitem_narrator](https://user-images.githubusercontent.com/22956130/40724290-5b707ea6-643e-11e8-8680-5a9bb5b78125.png)
![positionsize_treeitem_nvda](https://user-images.githubusercontent.com/22956130/40724291-5b9cf472-643e-11e8-8766-2ff5cb2c4819.png)

### Environment: 
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.2.1
